cicd/index.adoc

```diff
@@ -15,19 +15,15 @@ toc::[]
 
 [id="openshift-builds"]
 == OpenShift Builds
-OpenShift Builds provides you the following option to configure and run a build:
+OpenShift Builds provides you the following options to configure and run a build:
 
-// * Builds using Shipwright
-// +
-// An extensible build framework based on the Shipwright project, which you can use to build container images on an {product-title} cluster. You can build container images from source code and Dockerfile by using image build tools, such as Source-to-Image (S2I) and Buildah.
-// +
-// For more information, see link:https://docs.openshift.com/builds/latest/about/overview-openshift-builds.html[Overview of Builds].
-
-* Builds using BuildConfig
+* Builds using Shipwright is an extensible build framework based on the Shipwright project. You can use it to build container images on an {product-title} cluster. You can build container images from source code and Dockerfile by using image build tools, such as Source-to-Image (S2I) and Buildah.
 +
-A declarative build process to create cloud-native apps. You can define the build process in a YAML file that you use to create a `BuildConfig` object. This definition includes attributes such as build triggers, input parameters, and source code. When deployed, the `BuildConfig` object builds a runnable image and pushes the image to a container image registry. With the `BuildConfig` object, you can create a Docker, Source-to-image (S2I), or custom build.
+For more information, see link:https://docs.openshift.com/builds/latest/about/overview-openshift-builds.html[Overview of Builds].
+
+* Builds using BuildConfig is a declarative build process to create cloud-native apps. You can define the build process in a YAML file that you use to create a `BuildConfig` object. This definition includes attributes such as build triggers, input parameters, and source code. When deployed, the `BuildConfig` object builds a runnable image and pushes the image to a container image registry. With the `BuildConfig` object, you can create a Docker, Source-to-image (S2I), or custom build.
```

Comment:
Does `BuildConfig` need backticks and/or does it need a bit more description to clarify? Something like: Builds created using `BuildConfig` objects use a declarative build process to create cloud-native apps.